### DISCUSSION TITLE
Stable docker image

### DISCUSSION THREAD

**Discussion by hajo62** [bpatrik/pigallery2#418] [Q&A] [answered]
I see nighly builds but the stable version is 8 month old and the install notes still say not to use the LATEST tag.

Is there an easy way to get a newer running version which includes the German language?

Additional questions:

- Is there a way to limit selectable languages - i.e. to en and de?
- How to set German as default?
- How can the default value for `share valid days` be changed i.e. from 30 to 7?

**Comment by bpatrik** [comment, marked as answer]
Hi
I'm planning to finish a few more features and make a new release. Unfortunately no deadline promises on that. (but I except it to happen in 1-2 months)
In the meantime you can use the nightly. 


-     Is there a way to limit selectable languages - i.e. to en and de?
       - not at the moment. Also your browser should select it automatically 
-     How to set German as default?
      - your browser should select it based on its (or the OS language)
-     How can the default value for share valid days be changed i.e. from 30 to 7?
       - no way of doing so

**Comment by hajo62** [comment]
I took a nightly but, strange enough, I do not even see German as available language. See #428 

And regarding the other two topics: Might be an idea for an enhancement?
